tools/Vis_PV.py

```diff
@@ -0,0 +1,286 @@
+from paraview.simple import *
+import sys
+import yaml
+import numpy as np
+import math
+#### disable automatic camera reset on 'Show'                                                      
+paraview.simple._DisableFirstRenderCameraReset()
+
+# Set up the point of view (location, angle, etc.)
+
+def set_defaultCamera(renderView1):
+    '''
+    Set view and camera properties to a default 
+    specified below
+    '''
+    renderView1.ViewSize = [904, 501]
+    renderView1.AxesGrid = 'GridAxes3DActor'
+    renderView1.CenterOfRotation = [0.0, 0.0, -22.98529815673828]
+    renderView1.StereoType = 0
+    renderView1.CameraPosition = [-95.16305892510273, 754.3761479590994, 426.10491067642846]
+    renderView1.CameraFocalPoint = [43.11431452669897, 23.554652286151217, -46.78071794111179]
+    renderView1.CameraViewUp = [0.09790398117889891, -0.5275190119519493, 0.843882990999677]
+    renderView1.CameraParallelScale = 715.9367631158923
+    renderView1.Background = [0.0, 0.0, 0.0]
+    return renderView1
+
+def load_InputData():
+    '''
+    Read data from yaml input file
+    '''
+    Input_stream = open("InputVis_PV.yaml", 'r')
+    Input_data = yaml.load(Input_stream)
```

Comment:
I would add all the sanity checks here so they are in one place. It also catches mistakes early on.